CITATION.cff

```diff
@@ -0,0 +1,17 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Lupo Pasini"
+  given-names: "Massimiliano"
+  orcid: "https://orcid.org/0000-0002-4980-6924"
+- family-names: "Reshniak"
+  given-names: "Viktor"
+  orcid: "https://orcid.org/0000-0003-1545-4462"
+- family-names: "Stoyanov"
+  given-names: "Miroslav"
+  orcid: "https://orcid.org/0000-0002-8199-5577"
+title: "AADL: Anderson Accelerated Deep Learning"
```

Comment:
Good idea, but we should also add the license, e.g.,
```
license: BSD-3-Clause
```
See here: https://github.com/citation-file-format/ruby-cff